src/vs/workbench/contrib/externalUriOpener/common/configuration.ts

```diff
@@ -64,6 +64,19 @@ export const externalUriOpenersConfigurationNode: IConfigurationNode = {
 	}
 };
 
+export const defaultBrowserNode: IConfigurationNode = {
+	...workbenchConfigurationNodeBase,
+	properties: {
+		'workbench.defaultBrowser': {
+			type: 'string',
+			markdownDescription: nls.localize('defaultBrowser', "Configure the default browser to use for opening http or https links"),
+			defaultSnippets: [{
+				body: 'google-chrome'
+			}],
+		}
+	}
```

Comment:
I would argue the setting needs to be `restricted: true` so that it can be defined in a workspace, but only in trusted workspaces.

Reply:
After more thinking, this should probably be `ConfigurationScope.APPLICATION` and not be workspace overridable.